### DISCUSSION TITLE
Showcase Your First Project and Get Featured!

### DISCUSSION THREAD

**Discussion by OlaiwonAbdullahi** [community/community#138708] [New to GitHub]
**Hey GitHub community!** 👋

Are you new to GitHub and excited about your first project? We want to hear from you! Sharing your work is a great way to get feedback, connect with others, and learn.

**How to Participate:**

- Post a link to your repository in the comments below.

- Share a brief description of your project and what inspired you.
- Mention any challenges or unique features you encountered.
- Tell us a little about yourself. What brought you to GitHub, and what are your goals?
*Why Share?*

- Get valuable feedback from experienced developers.
- Gain visibility and connect with like-minded individuals.
- Learn and grow through constructive suggestions and support.
- Earn a dedicated spotlight post in the New to GitHub community.
**Remember:** Every project, no matter how big or small, is a step in your development journey. Don't be shy—share your hard work with us!

Let's celebrate your first steps and create a supportive community together. 🚀

Happy coding! 💻

**Comment by sctech-tr** [comment]
# upm
<img src="https://raw.githubusercontent.com/sctech-tr/upm/main/upm.png" width="250" height="250">
upm is a universal package manager that automatically detects your primary package manager and runs the appropriate command, so you don't have to find a way to detect the package manager within code. it's written in bash (for linux and macos) and in powershell (for windows)

features:
- supports snap and flatpak
- windows support with package managers (winget, choco and scoop)

fun fact: i accidentally found the github font while making the logo :D

check it out!

https://github.com/sctech-tr/upm

**Reply by OlaiwonAbdullahi** [reply]
Wow what a nice Project wont you mind connecting with you?

**Reply by sctech-tr** [reply]
send an email here: https://sctech.mooo.com/send.html
or open an issue here: https://github.com/sctech-tr/sctech-tr/issues

**Comment by typed-sigterm** [comment]
https://github.com/typed-sigterm/ex-caller - a simple random call tool

Preview:
- [dev] https://main--ex-caller.netlify.app/ (full English tranlation except tour)
- [production] http://ex-caller.by-ts.top/ (English tranlation not published yet)

It was initially designed to facilitate teachers to randomly select people to answer questions during class.

So far, the biggest challenge in development is:
- the performance and pitfalls of web stack
- the inability to simulate release to test whether tauri updater is working

The reason I'm here to share is... hoping to get some inspiration? maybe

**Comment by NicholasGannon** [comment]
## Font Resources for Designers + Developers
https://github.com/NicholasGannon/Font-Resources
- The goal of this project is to provide a curated catalog of **high quality font/typography services and collections** that have practical use.
- My main inspiration behind this project was the general difficulty that users face when trying to **enhance their projects through font.**
- It's somewhat easy to find font resources online, but I find it **particularly challenging** to find ones that are reliable and easy-to-implement.

### I'd really appreciate people checking out this repo and giving it a star! Thank you!

**Comment by ghostinhershell** [comment]
Hey there @OlaiwonAbdullahi 👋🏾 

Did you guys know that our GitHub community is actually running a similar campaign right now? Check it out here! https://github.com/orgs/community/discussions/137746 🚀

**Reply by OlaiwonAbdullahi** [reply]
Really

**Comment by pengpengyi92** [comment]
## dsh-quant — an agent-native quantitative research toolkit 🐳

**Repo:** https://github.com/pengpengyi92/dsh-quant · **npm:** `dsh-quant`

I built a plugin-based quant research OS on top of DeepSeek Harness: **46 tools across 6 domains** (data / alpha / ML / risk / execution), with free market data for crypto, A-shares, and US stocks, a channel knowledge base, and multi-asset parallel research pipelines.

**What inspired me:** I kept seeing the same gap — quant research workflows scattered across scripts, notebooks, and silos. I wanted everything (data → factors → ML → risk → execution) to live in one pipeline where both humans and AI agents can contribute. The design rule is "everything is a plugin": five pluggable slots, users bring their own.

**Unique features:**
- AI-native by design: tools speak a unified schema DSL, agents read `AGENTS.md` and can contribute plugins themselves
- Free market data for 3 markets (Binance/OKX, Sina/Tencent A-shares, Yahoo global) — no API keys
- 215 tests, 60+ automated releases, installable via `npm i dsh-quant`
- Live demo: https://dsh-quant-ui.pages.dev (real BTC backtest, playable in browser)

**Challenges:** keeping method-open while secrets stay internal, and making sure the pipeline closes the loop from raw data to execution.

Happy to get feedback from this community! 🐋

**Comment by pengpengyi92** [comment]
**Update 2026-08-31 — dsh-quant v0.90.0 is now 59 tools across 6 domains** 🐳

Since the v0.14 release, the toolkit grew from 46 to **59 `quant_*` tools**:
- New risk suite: historical VaR / CVaR / drawdown / Beta / Jensen Alpha with Kupiec POF backtest
- K-line weekly/monthly resampling, one-shot Markdown research reports
- FICC coverage: rates, credit bonds, FX, commodity/gold guides + 162-file quant-history knowledge base
- Multi-exchange fallback chain (Binance geo-block → auto-switch OKX, tested)
- 215 unit tests, 0 runtime dependencies, ~18.8k npm downloads/month

#quant #opensource #deepseek-harness #dsh-plugin #ai-native

Repo: https://github.com/pengpengyi92/dsh-quant · Site: https://dsh-quant-site.pages.dev

**Comment by pengpengyi92** [comment]
**Update 2026-08-31 — dsh-quant v0.90.0 is now 59 tools across 6 domains** 🐳

Since the v0.14 release, the toolkit grew from 46 to **59 `quant_*` tools**:
- New risk suite: historical VaR / CVaR / drawdown / Beta / Jensen Alpha with Kupiec POF backtest
- K-line weekly/monthly resampling, one-shot Markdown research reports
- FICC coverage: rates, credit bonds, FX, commodity/gold guides + 162-file quant-history knowledge base
- Multi-exchange fallback chain (Binance geo-block → auto-switch OKX, tested)
- 215 unit tests, 0 runtime dependencies, ~18.8k npm downloads/month

#quant #opensource #deepseek-harness #dsh-plugin #ai-native

Repo: https://github.com/pengpengyi92/dsh-quant · Site: https://dsh-quant-site.pages.dev
